teapp/src/main/java/com/techelevator/archive/SmsSender.java

```diff
@@ -0,0 +1,54 @@
+package com.techelevator.archive;
+// Install the Java helper library from twilio.com/docs/libraries/java
+import com.twilio.Twilio;
+import com.twilio.rest.api.v2010.account.Message;
+import com.twilio.type.PhoneNumber;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class SmsSender {
+    // Find your Account Sid and Auth Token at twilio.com/console
+    public static String ACCOUNT_SID;
+    public static String AUTH_TOKEN;
+
+    public static void sendMessage(String msg, String phone) {
+        loadProperties();
+        Twilio.init(ACCOUNT_SID, AUTH_TOKEN);
+
+        Message message = Message
+                .creator(new PhoneNumber(phone), // to
+                        new PhoneNumber("+18317775889"), // from
+                        msg)
+                .create();
+
+        System.out.println(msg);
+    }
+
+    private static void loadProperties() {
+        Properties prop = new Properties();
+        InputStream input = null;
+
+        try {
+            input = new FileInputStream("teapp\\src\\main\\resources\\twilioConfig.properties");
```

Comment:
I'd suspect there's an easier way to do this. Usually if a library relies on a config file it can autoload the config file from a standard location rather than having to do this manually.